src/modules/testimonials/testimonial_entity.ts

```diff
@@ -1,26 +1,40 @@
+import { Query } from '../../helpers/types';
+
 export namespace Testimonial {
 
   export interface Struct {
     id?: string
     caption: string
-    created_at: Date
+    created_at?: Date
     created_by: string
   }
 
   export interface TestimonialList {
     id: string
     caption: string
-    type: string
     user: {
       id: string
       name: string
+      type: string
       description: string
       avatar: string
     }
   }
 
+  export interface RequestQuery extends Query {
+    per_page: string,
+    current_page: string
+  }
+
   export interface ResponseFindAll {
     data: TestimonialList[]
-    meta: Object
+    meta: {
```

Comment:
please considering this. 
for meta object, this code can make reusable interface in helpers module or other at the project because meta object can use for features that use pagination like list ,..
cmiwiw cc @ayocodingit

Reply:
oh i see for meta pagination, we will helper global for mapping object, thanks :smile: @sandisunandar99